.github/workflows/codeql.yml

```diff
@@ -36,7 +35,7 @@ jobs:
         queries: security-and-quality
 
     # build between init and analyze ...
-    - name: Configure and Compile boreas
+    - name: Configure and Compile openvas-smb
```

Comment:
can you change github/codeql-action/init@v3 and github/codeql-action/analyze@v3 ?